request.go

```diff
@@ -82,7 +82,9 @@ func (req *Request) applyHeader() {
 }
 
 func (req *Request) isSuccessfulPost() bool {
-	return req.RawResponse.StatusCode == http.StatusOK || req.RawResponse.StatusCode == http.StatusCreated
+	return req.RawResponse.StatusCode == http.StatusOK ||
+		req.RawResponse.StatusCode == http.StatusCreated ||
+		req.RawResponse.StatusCode == http.StatusNoContent
```

Comment:
Das sollte im Fall von `PostForBody(…)` nicht als erfolg gewertet werden. Der Server sollte in diesem Fall keinen Body senden...

Reply:
Interessanterweise wird diese Methode nicht für PostForBody benutzt. Aber für PutForBody. Ich passe das mal an

Reply:
Ich hab auch noch ein paar andere Fälle die mir falsch vorkammen angepasst.